code/modules/spells/targeted/absorb.dm

```diff
@@ -39,6 +39,7 @@
 					for(var/spell/targetspell in C.spell_list)
 						canAbsorb = TRUE
 						for(var/spell/holderspell in L.spell_list)
+							if(targetspell.user_type == USER_TYPE_VAMPIRE || targetspell.user_type == USER_TYPE_CULT  || targetspell.user_type == USER_TYPE_MALFAI || targetspell.user_type == USER_TYPE_GENETIC)
```

Comment:
just make it check for user_type == USER_TYPE_WIZARD

Reply:
there are spells that should be absorbed that fall under other types like user_type_spellbook and it's like this in case a new type gets added